### PR TITLE
Add README in chinese

### PR DESCRIPTION
**Pull request by xxx7xxxx**

### CODE REVIEW

File: README.zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+1. [EaseAgent](https://github.com/megaease/easeagent)
+2. [Easegress](https://github.com/megaease/easegress)
+
+## 6. 快速启动
+
+### 6.1 环境要求
+
+- Linux内核版本4.15以上
+- Kubernetes版本1.18以上
+- MySQL版本14.14+
```
Comment (zhao-kun) [inline README.zh-CN.md:131]: Invalid mysql version !(should fix it in English version)

File: README.zh-CN.md [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+    kind: Proxy
+    mainPool:
+      servers:
+      - url: http://{node1_of_k8s_cluster}:{port_exposed_by_ingress_service}
+      - url: http://{node2_of_k8s_cluster}:{port_exposed_by_ingress_service}
+      loadBalance:
+        policy: roundRobin
+```
+
+根据你的环境改变`{}`中的内容，并通过Easegress客户端命令工具`egctl`应用它。
```
Comment (zhao-kun) [inline README.zh-CN.md:273]: `应用它` -> `生效它`

File: README.zh-CN.md (hunk 1 above)
Comment (xxx7xxxx) [reply README.zh-CN.md]: Updated with 5.7+

File: README.zh-CN.md (hunk 2 above)
Comment (xxx7xxxx) [reply README.zh-CN.md]: done

**Review by zhao-kun — APPROVED** [review]

**Pull request merged**